sinks/attribution/csv.go

```diff
@@ -0,0 +1,54 @@
+package attribution
+
+import (
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"strconv"
+)
+
+// ErrUnknownSchemaVersion is an error returned when the config specifies a
+// schema version that does not exist
+var ErrUnknownSchemaVersion = errors.New("specified schema version does not exist")
```

Comment:
`const`?

Reply:
const can only be string, char, bool, or numerical